emscripten/trackerMM.cpp

```diff
@@ -25,11 +25,17 @@ void TrackerMM::setup(string camParamFile, string multiFile, int maxImagePattern
 
   // Initialize a Single Marker Tracker with
   // Camera and near and far clipping values for the OpenGL projection matrix
-  if (!tracker->init(camParamFile.c_str(), multiFile.c_str(), 1.0f, 1000.0f)) {
+  /*if (!tracker->init(camParamFile.c_str(), multiFile.c_str(), 1.0f, 1000.0f)) {
```

Comment:
i had troble loading both files, now i will develop my init function:
in this commit only load the camera but we can load the multi file  and calib file separately.